osc-bsu-csi-driver/values.yaml

```diff
@@ -32,7 +32,7 @@ backoff:
 
 sidecars:
   provisionerImage:
-    repository: k8s.gcr.io/sig-storage/csi-provisioner
+    repository: registry.k8s.io/sig-storage/csi-provisioner
```

Comment:
This explain why 
https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/.